package.json

```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "chai": "~2.3.0",
     "coffee-script": "~1.9.2",
-    "gulp": "~3.8.11",
+    "gulp": "~4.0.2",
```

Comment:
Required for Node 12 compatibility. Otherwise, the error was:
`ReferenceError: primordials is not defined`